Examples/Google/Language/Sources/main.swift

```diff
@@ -0,0 +1,182 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOTE: This file is automatically-generated!
+import Foundation
+import Dispatch
+import OAuth2
+import GoogleAPIRuntime
+import Commander
+
+let CLIENT_CREDENTIALS = "language.json"
+let TOKEN = "language.json"
+
+func main() throws {
+  let scopes = [
+    "https://www.googleapis.com/auth/cloud-language",
+    "https://www.googleapis.com/auth/cloud-platform"]
+
+  guard let tokenProvider = BrowserTokenProvider(credentials:CLIENT_CREDENTIALS, token:TOKEN) else {
+    return
+  }
+  let language = try Language(tokenProvider:tokenProvider)
+
+  let group = Group {
+    $0.command("login", description:"Log in with browser-based authentication.") {
+      try tokenProvider.signIn(scopes:scopes)
+      try tokenProvider.saveToken(TOKEN)
+    }
+
+    $0.command(
+      "documents.analyzeEntities",
```

Comment:
WRT the camelCase method names, have you thought about massaging it into kebab-case or snake_case? capitalization is easy to mess up & hard to debug imo. But if completion is available then this isn't really a problem.

In looking at the disco, I understand the desire to stay as close to the definition as possible too.

Reply:
I need to review this code against our internal Swift style guide, but I believe that it's consistent with [Ray Wenderlich's Swift Style guide](https://github.com/raywenderlich/swift-style-guide#naming) which prefers camel case over snake case.